packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -68,6 +77,17 @@ class Layout extends React.Component {
             />
           </nav>
           <main id="main-container" className={cx('main-container')}>
+            {siteAlert.map(
```

Comment:
what if we do not provide siteAlert property.

Reply:
If I don't provide siteAlert property it throws error. we need to pass [] atleast .